src/Tribe/Views/V2/View.php

```diff
@@ -1393,7 +1393,16 @@ protected function setup_template_vars() {
 			? Dates::build_date_object( $event_date )->format( Dates::DBDATEFORMAT )
 			: false;
 
-		$template_vars = [
+		/*
+		 * Some plugins, like WooCommerce, will modify the UID of logged out users; avoid that filtering here.
+		 *
+		 * @see TEC-3579
+		 */
+		$rest_nonce = tribe_without_filters( [ 'nonce_user_logged_out' ], static function () {
+			return wp_create_nonce( 'wp_rest' );
+		} );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 9a81805eb2615ddd4d5215a5db136b01</sub>